src/probnum/linops/_linear_operator.py

```diff
@@ -172,6 +174,97 @@ def __call__(self, x: np.ndarray, axis: Optional[int] = None) -> np.ndarray:
         else:
             raise ValueError("The operand must be at least one dimensional.")
 
+    def solve(self, b: ArrayLike) -> np.ndarray:
+        """Solves linear systems :code:`A @ x = b`, where :code:`b` is either a vector
+        or a (stack of) matrices.
+
+        This method behaves exactly like :code:`A.inv() @ b`.
```

Comment:
Do we want to guarantee this? This is generally not the case for numpy arrays and will also not be the case anymore once we allow the user to choose a linear solver. I suggest not promising this in the docstring.
```suggestion
```

Reply:
I changed the docstring to communicate what I actually meant with this line in the first place.